bin/setup-on-cluster

```diff
@@ -1,39 +1,6 @@
 #!/bin/sh
```

Comment:
Can we please get rid of this script completely?

Reply:
removed bin/setup-on-cluster